lib/que/sequel/model.spec.rb

```diff
@@ -7,7 +7,7 @@
     require "que/sequel/model"
   end
 
-  def enqueue_job(*args)
+  ruby2_keywords def enqueue_job(*args)
```

Comment:
Yes, but I think we should change all these spec helper methods to accept a callable that makes the r queue call directly rather than passing the args and kwargs through, then we don’t need it.

Reply:
I started down that path, but it ended up making the tests look a lot more complex. The usage of `enqueue_job` is relatively simple and we control it, so it seems ok to have a helper to me. I did notice that the model specs for Sequel and ActiveRecord had diverged, so I made them consistent.